tests/utest/zap/test_zap_cli.py

```diff
@@ -0,0 +1,118 @@
+from ..helpers import RESOURCES_PATH
+from oxygen.oxygen import OxygenCLI
+from robot.running.model import TestSuite
+from unittest import TestCase
+from unittest.mock import ANY, create_autospec, Mock, patch
+
+
+import sys
+
+
+class TestOxygenZapCLI(TestCase):
+    zap_xml = str(RESOURCES_PATH / 'zap' / 'zap.xml')
+    base_args = ['oxygen', 'oxygen.zap', zap_xml]
+
+    def setUp(self):
+        self.cli = OxygenCLI()
+        self.handler = self.cli._handlers['oxygen.zap']
+
+    def test_cli(self):
+        self.assertEqual(
+            self.handler.cli(),
+            {('--accepted-risk-level',): {
+                'help': 'Set accepted risk level',
+                'type': int},
+             ('--required-confidence-level',): {
+                 'help': 'Set required confidence level',
+                 'type': int},
+             ('result_file',): {}})
+
+    @patch('oxygen.oxygen.RobotInterface')
+    @patch.object(sys, 'argv', base_args)
+    def test_cli_run(self, mock_robot_iface):
+        expected_suite = create_autospec(TestSuite)
+        mock = Mock()
+        mock.running.build_suite = Mock(return_value=expected_suite)
+        mock_robot_iface.return_value = mock
+
+        self.cli.run()
+
+        self.assertEqual(self.handler._config['accepted_risk_level'], 2)
+        self.assertEqual(self.handler._config['required_confidence_level'], 1)
+
+        mock.running.build_suite.assert_called_once()
+
+        expected_suite.run.assert_called_once_with(
+            output=str(RESOURCES_PATH / 'zap' / 'zap_robot_output.xml'),
+            log=None,
+            report=None,
+            stdout=ANY
+        )
```

Comment:
What's the point of these verifications? As far as I can see, the test is not interested in these details -- it seems they are just copy-paste?